src/tests/dispatch-payload.test.ts

```diff
@@ -116,10 +117,17 @@ Deno.test("DispatchPayload function file compatibility tests", async (t) => {
   const fixturesDir = `${__dirname}/fixtures`;
   Deno.chdir(fixturesDir);
   const functionsDir = `${fixturesDir}/functions`;
+  mockFetch.install(); // mock out calls to fetch
 
   await t.step(
     "return from provided file",
```

Comment:
I saw this test and wasn't sure the reason for it? I think we are covered here from other tests? Do we need to keep this entire `Deno.test` and its two sub-tests? I am leaning towards this can be removed.